source/Core/Threads/GUIThread.cpp

```diff
@@ -172,7 +172,7 @@ static void gui_drawBatteryIcon() {
 }
 static void gui_solderingTempAdjust() {
   TickType_t lastChange              = xTaskGetTickCount();
-  currentTempTargetDegC              = 0; // Turn off header while adjusting temp
```

Comment:
This is done deliberately because the irons heat so fast compared to their cooldown time. Complaints came from if you want to lower temperature it will heat up to old temperature in the time it takes to adjust.

If you want something like this, we should live track the temperature as its adjusted?

Reply:
Sorry, I was not aware of the background you described. 😅

But I am not capable of pulling of what you proposed instead.